packages/flutter/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [4.0.0](https://github.com/parse-community/Parse-SDK-Flutter/compare/flutter-3.1.3...flutter-4.0.0) (2023-03-09)
+
+### Breaking change
+
+* Change the source file name ([#845](https://github.com/parse-community/Parse-SDK-Flutter/issues/845))
+  Rather than importing `parse_server_sdk.dart` now you need to import `parse_server_sdk_flutter.dart`
+
```

Comment:
```suggestion
## Unreleased (master branch)

### BREAKING CHANGES

* The source file name of the Flutter package has changed; to import the Flutter package use `parse_server_sdk_flutter.dart` instead of `parse_server_sdk.dart` ([#846](https://github.com/parse-community/Parse-SDK-Flutter/pull/846))

### Features

* Rename flutter package source file to `parse_server_sdk_flutter.dart` ([#846](https://github.com/parse-community/Parse-SDK-Flutter/pull/846))

```